forge-gui/res/cardsfolder/l/loose_in_the_park.txt

```diff
@@ -0,0 +1,13 @@
+Name:Loose in the Park
+ManaCost:1 G
+Types:Enchantment Aura
+K:Enchant land
+A:SP$ Attach | Cost$ 1 G | ValidTgts$ Land | AILogic$ Pump
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraft | TriggerZones$ Battlefield | TriggerDescription$ When CARDNAME enters the battlefield, draw a card, then draft a card from Loose in the Park’s spellbook and exile it.
+SVar:TrigDraft:DB$ Draft | Spellbook$ Bristling Boar,Enraged Ceratok,Exuberant Wolfbear,Gaea's Protector,Master Symmetrist,Ornery Dilophosaur,Overgrown Armasaur,Predatory Wurm,Prized Unicorn,Sabertooth Mauler,Spike-Tailed Ceratops,Spore Crawler,Thrashing Brontodon,Wardscale Crocodile,World Shaper | Zone$ Exile | RememberDrafted$ True | SubAbility$ DBEffect    
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ Play | ExileOnMoved$ Exile 
+SVar:Play:Mode$ Continuous | MayPlay$ False | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile
+A:AB$ Animate | Cost$ 3 | Defined$ Enchanted | Keywords$ Haste | Types$ Land | SubAbility$ DDCopy | TriggerDescription$ {3}: Enchanted land becomes a copy of the exiled card until end of turn and gains haste. It's still a land.
+SVar:DDCopy:DB$ Clone | Defined$ Remembered | CloneTarget$ Enchanted | Types$ Land | Duration$ UntilEndOfTurn
```

Comment:
Only CloneEffect with `PumpKeywords$` will be enough

Reply:
I've added the card draw effect and removed DBE effect. I've tested the new script and it seemed to work 

> Only CloneEffect with `PumpKeywords$` will be enough

Hmm, I've tried A:AB$ Clone | Cost$ 3 | Defined$ Remembered | CloneTarget$ Enchanted | Keywords$ Haste | Types$ Land  | TriggerDescription$ {3}: Enchanted land becomes a copy of the exiled card until end of turn and gains haste. It's still a land.

This didn't work since the aura falls off when you activate the clone ability

Reply:
`A:AB$ Clone | Cost$ 3 | Defined$ Remembered | CloneTarget$ Enchanted | PumpKeywords$ Haste | AddTypes$ Land | Duration$ UntilEndOfTurn | SpellDescription$ {3}: Enchanted land becomes a copy of the exiled card until end of turn and gains haste. It's still a land.`